Snooper/src/main/java/com/prateekj/snooper/networksnooper/database/SnooperRepo.java

```diff
@@ -130,9 +140,16 @@ public HttpCallRecord findById(long id) {
   }
 
   public void deleteAll() {
-    SQLiteDatabase database = snooperDbHelper.getWritableDatabase();
-    database.delete(HTTP_CALL_RECORD_TABLE_NAME, null, null);
-    database.close();
+    SQLiteDatabase database = dbWriteHelper.getWritableDatabase();
+    try {
+      database.beginTransaction();
+      database.delete(HTTP_CALL_RECORD_TABLE_NAME, null, null);
+      database.setTransactionSuccessful();
+    }
+    finally {
+      database.endTransaction();
+      database.close();
+    }
```

Comment:
LGTM, Just make sure `SnooperRepoTest.java` is passing after this change.

Thanks, :)

Reply:
I've run all of the tests and they pass locally.